experiments/storm/src/main/scala/io/gearpump/experiments/storm/GearpumpThriftServer.scala

```diff
@@ -20,32 +20,31 @@ package io.gearpump.experiments.storm
 
 import java.nio.ByteBuffer
 import java.util.concurrent.TimeUnit
+import java.util.{HashMap => JHashMap, Map => JMap}
 
 import akka.actor.ActorRef
+import backtype.storm.Config
 import backtype.storm.generated._
+import backtype.storm.security.auth.{ThriftConnectionType, ThriftServer}
+import backtype.storm.utils.Utils
 import io.gearpump.experiments.storm.Commands.{GetClusterInfo, _}
 import io.gearpump.util.ActorUtil.askActor
-import org.apache.thrift7.protocol.TBinaryProtocol
-import org.apache.thrift7.server.{THsHaServer, TServer}
-import org.apache.thrift7.transport.TNonblockingServerSocket
 
 import scala.concurrent.Await
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
+import scala.language.postfixOps
 
 object GearpumpThriftServer {
-  val THRIFT_PORT = 6627
-  val THRIFT_THREADS = 64
-  val THRIFT_MAX_BUFFER_SIZE = 1048576
+  val THRIFT_PORT = 6628
```

Comment:
This is important especially we want to launch a storm jar by submiting in web UI.